api/src/domain/user/taxEncryptionDecryptionFactory.ts

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const AWSCrypto = require("@aws-crypto/client-node");
+import { fromBase64, toBase64 } from "@aws-sdk/util-base64-browser";
+import { TextDecoder } from "node:util";
+import { EncryptionDecryptionClient } from "../types";
+
+type context = {
+  stage: string;
+  purpose: string;
+  origin: string;
+};
+
+export const TaxEncryptionDecryptionFactory = (
+  generatorKeyId: string,
+  keyIds: string[],
```

Comment:
why does this parameter exist? It looks like we're never using it in the app.ts file?